### PR TITLE
Fixes null-named painting issue

### PR DESCRIPTION
**Pull request by CoffeeDragon16**
## About The Pull Request

There was an issue where you could name paintings nothing, which would cause issues with persistency. This was because the stripped_input didn't actually check that you put something in, so the PR adds that check. It now ensures, both when saved and loaded, that there is a title. If not, it sets them to the default 'Untitled Artwork'

## Why It's Good For The Game

Bugs bad.

## Changelog
:cl:
fix: Paintings are now unable to have no name.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/artstuff.dm [hunk 1]
```diff
@@ -185,6 +185,8 @@
 
 /obj/item/canvas/proc/try_rename(mob/user)
 	var/new_name = stripped_input(user,"What do you want to name the painting?")
+	if(!new_name)
+		new_name = "Untitled Artwork"
 	if(new_name != painting_name && new_name && user.canUseTopic(src,BE_CLOSE))
```
Comment (AnturK) [inline code/game/objects/structures/artstuff.dm:190]: But it's checked here ?

Comment (CoffeeDragon16) [reply code/game/objects/structures/artstuff.dm:190]: That doesn't stop you from hanging it with no name which creates the error, if I'm understanding it right. This is actually setting a name for when there's no name.

Comment (AnturK) [reply code/game/objects/structures/artstuff.dm:190]: There's a default name set above.

Comment (CoffeeDragon16) [reply code/game/objects/structures/artstuff.dm:190]: Yeah, but doesn't using the input change that?

Comment (CoffeeDragon16) [reply code/game/objects/structures/artstuff.dm]: Removed that, put it in the save and load. This should prevent new NULL names from popping up, and get rid of the old ones.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**